frontends/web/src/routes/device/components/skipfortesting.tsx

```diff
@@ -14,51 +14,40 @@
  * limitations under the License.
  */
 
-import React, { Component } from 'react';
+import React, { useState } from 'react';
+import { testRegister } from '../../../api/account';
 import { Button } from '../../../components/forms';
 import { PasswordSingleInput } from '../../../components/password';
-import { apiPost } from '../../../utils/request';
 
-interface SkipForTestingProps {
+type TProps = {
     show: boolean;
 }
```

Comment:
I wonder if we need this at all? 🤔

Reply:
good catch. this seems to be a left over from when it used show prop https://github.com/digitalbitbox/bitbox-wallet-app/blob/71a45aa2c6566bc1a50433a56750e7ee4f9778e2/frontends/web/src/routes/device/waiting.tsx#L60